volestipy/test1.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from volestipy import HPolytope
+
+
+if __name__ == "__main__":
+    dim = 2
+    A = np.zeros((2*dim, dim), dtype=np.float)
+    A[0:dim] = np.eye(dim)
+    A[dim:] -=  np.eye(dim,dim, dtype=np.float)
+    b = np.ones(2*dim, dtype=np.float)
+
+
+    p = HPolytope(A,b)
+    print("The polytope:")
+    print("Dimensions: {}".format(p.dimensions))
+    print("A matrix:")
+    print(p.A)
+    print("b vector:")
+    print(p.b)
+
+    volume = p.compute_volume(walk_len=5, epsilon=0.05, method="gaussian_annealing", seed=42)
```

Comment:
just being curious here, why 42 and not some value from the cpu clock ?

Reply:
42 was just for fixing the seed while testing. I changed the default seed in the volestipy.pyx file to use the current time for the seed. Omitting this argument in p.compute volume while now default to using the current time.